docs/How-To/chain_001_faucet.md

```diff
@@ -0,0 +1,16 @@
+
+# Get Testnet Tokens (Faucet)
+
+The Cosmos Cash native token denom is `cash`.  Assuming that a keypair has been created with the follwing command:
+
+```sh
+cosmos-cashd keys add alice
+```
+
+to obtain obtain `cash` tokens for the alice account from the testnet faucet use the following command:
```

Comment:
```suggestion
To obtain obtain `cash` tokens for the alice account from the testnet faucet, use the following command:
```

Reply:
let's use sentence grammar, and add that missing comma; the colon before the command is helpful for readers to know a command is next